src/libraries/Microsoft.PowerFx.Core/Public/Config/FormulaValueSerializerSettings.cs

```diff
@@ -12,5 +12,7 @@ namespace Microsoft.PowerFx
     /// </summary>
     public class FormulaValueSerializerSettings
     {
+        // Provides a simpler version for some types.
+        public bool Compact { get; set; }
```

Comment:
>Compact [](http://example.com/codeflow?start=20&length=7)

Suggestion: `UseCompactRepresentation`. Better explains what this flag is supposed to do